File: scratch/check_db_products.py
import sqlite3

conn = sqlite3.connect('prisma/cms.db')
cursor = conn.cursor()

try:
    cursor.execute("SELECT id, slug, name FROM Product;")
    rows = cursor.fetchall()
    print(f"Total products in SQLite DB (prisma/cms.db): {len(rows)}")
    for i, r in enumerate(rows, 1):
        print(f"{i}. [{r[0]}] {r[1]}")
except Exception as e:
    print(f"Error querying SQLite DB: {e}")
finally:
    conn.close()

File: scripts/check-db-products.mjs
import { PrismaClient } from '@prisma/client';

const prisma = new PrismaClient();

async function main() {
  const count = await prisma.product.count();
  console.log('Total products currently in Supabase:', count);
  const products = await prisma.product.findMany({
    select: { id: true, name: true, slug: true, tagline: true, category: true, price: true }
  });
  console.log(JSON.stringify(products, null, 2));
}

main()
  .catch(console.error)
  .finally(() => prisma.$disconnect());
